test/__snapshots__/index.test.js.snap

```diff
@@ -1642,15 +1649,15 @@ Build \\"mangleJsonFile\\" to dist:
 103 B: mangle-json-file.js.gz
 89 B: mangle-json-file.js.br
 103 B: mangle-json-file.esm.js.gz
-95 B: mangle-json-file.esm.js.br
+84 B: mangle-json-file.esm.js.br
 198 B: mangle-json-file.umd.js.gz
 164 B: mangle-json-file.umd.js.br"
 `;
 
 exports[`fixtures build mangle-json-file with microbundle 2`] = `6`;
 
 exports[`fixtures build mangle-json-file with microbundle 3`] = `
-"var o={prop1:1,__p2:2};export default function(){return console.log(o.prop1),console.log(o.__p2),o}
+"var a={prop1:1,_prop2:2};export default function(){console.log(a.prop1);console.log(a._prop2);return a}
```

Comment:
This looks like a bug or issue to resolve before merging, the `mangled` value isn't being used.

Reply:
https://github.com/developit/microbundle/wiki/mangle.json#for-manglejson-implementors